Tests/SwiftStdlibTests/CodableExtensionsTests.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CodableExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Mustafa GUNES on 16.01.2020.
+//  Copyright © 2020 SwifterSwift
+//
+
+import XCTest
+
+@testable import SwifterSwift
+
+private struct City: Codable {
+    var cityId: Int
+    var name: String
+    var photoUrl: String
+
+    private enum CodingKeys: String, CodingKey {
+        case cityId = "id"
```

Comment:
why `cityId = "id"`, just make it `id`

Reply:
@omaralbeik I named it that way because of the swiftlint alert. But right now I would fix the problems :)

Reply:
> @omaralbeik I named it that way because of the swiftlint alert. But right now I would fix the problems :)

In this case, you can tell swiftlint to ignore the rule for this line

```swiftlint
// swiftlint:disable:next identifier_name
case id
```